front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -140,48 +137,28 @@ const TopicsFilter = memo<Props>(
         >
           <Box>
             <Box className="e2e-topics-filters" aria-live="polite">
-              {topicsWithIdeas
-                .filter((topic) => !isError(topic))
-                .map((topic: ITopicData, index) => {
-                  const postCount = get(
-                    filterCounts,
-                    `topic_id.${topic.id}`,
-                    0
-                  );
-
-                  const topicSelected = selectedTopicIds?.includes(topic.id);
-
-                  if (!showFullList && index > 4) {
-                    // Only show the first 5 topics by default
-                    return null;
-                  }
-
-                  return (
-                    <Topic
-                      key={topic.id}
-                      data-id={topic.id}
-                      onMouseDown={removeFocusAfterMouseClick}
-                      onClick={handleOnClick}
-                      className={`e2e-topic ${topicSelected ? 'selected' : ''}`}
-                      style={{ display: 'flex' }}
-                    >
-                      <Box maxWidth="90%">
```

Comment:
Was there a reason for this `maxWidth`? I removed it and everything seems fine.

Reply:
From an old commit I think - thanks for catching it :)